index.html

```diff
@@ -262,5 +273,27 @@ <h2>Skills</h2>
         </section>
       </section>
     </main>
+    <footer class="footer-section">
+      <div class="footer-text flex">
+        <h2 class="footer-title">
+          I'm always interested in hearing about new projects, 
+          so if you'd like to chat please get in
+        </h2>
+      </div>
+      <div class="footer-items grid">
+        <div class="form-con">
+          <form class="form-item flex" action="https://formspree.io/f/mnqwkpvo" method="POST">
+            <input type="text" name="user_name" placeholder="Full name" maxlength="30" required>
+            <hr class="form-br">
+            <input type="email" name="user_email" placeholder="Email address" maxlength="30" required>
+            <hr class="form-br">
+            <textarea name="user_message" maxlength="500" required>Enter text here</textarea>
+            <button class="footer-btn special" type="submit">Get in touch</button>
```

Comment:
- [ ] kindly I would recommend you to put button interactions on this submit button to make your form more interactive hence displaying professionalism in what you do. examples of interactions(hover, focus, etc)